sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/src/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents.csproj

```diff
@@ -10,6 +10,7 @@
 		<PackageReference Include="Microsoft.Azure.WebJobs" />
 		<PackageReference Include="Microsoft.AspNetCore.Http" />
 		<PackageReference Include="System.IdentityModel.Tokens.Jwt" />
+		<PackageReference Include="System.Net.Http.Json" VersionOverride="7.0.0" />
```

Comment:
I would just confirm with the review board if this is okay.

Reply:
Verified that this how they are doing it in other libraries. I don't think it will be an issue. 

Regardless, can you please connect me with the review board? I will run it by them.